beacon-chain/rpc/endpoints.go

```diff
@@ -983,6 +983,16 @@ func (s *Service) prysmBeaconEndpoints(
 			handler: server.GetIndividualVotes,
 			methods: []string{http.MethodPost},
 		},
+		{
+			template: "/prysm/v1/beacon/chain_head",
+			name:     namespace + ".GetChainHead",
+			middleware: []mux.MiddlewareFunc{
+				middleware.ContentTypeHandler([]string{api.JsonMediaType}),
```

Comment:
you don't need the content type handler for get requests

Reply:
Nice catch, it's a copy paste error